frappe/website/doctype/discussion_reply/discussion_reply.py

```diff
@@ -59,4 +59,6 @@ def after_delete(self):
 
 @frappe.whitelist()
 def delete_message(reply_name):
-	frappe.delete_doc("Discussion Reply", reply_name, ignore_permissions=True)
+	owner = frappe.db.get_value("Discussion Reply", reply_name, "owner")
+	if owner == frappe.session.user:
```

Comment:
This could've been done handled with our standard permission model too right? Arguably better, given how privileged users cannot delete Discussion Reply docs from this API but can use a couple others to do the same 😅

I think this API is an inconsistency. We can get rid of it and use standard DELETE resource API call